lib/cluster_ht_item.rb

```diff
@@ -2,118 +2,72 @@
 
 require "cluster"
 require "reclusterer"
-require "cluster_error"
+require "cluster_getter"
+require "retryable"
 
 # Services for batch loading HT items
-#
-# rubocop:disable Metrics/ClassLength
-# Much of this esp. around transactions and retry should be extracted to common
-# functionality shared by the other Cluster classes; remove this disabled cop
-# after doing so.
 class ClusterHtItem
 
-  MAX_RETRIES=5
-  # Not constantized by mongo ge
-  MONGO_DUPLICATE_KEY_ERROR=11_000
+  def initialize(*ht_items)
+    @ht_items = ht_items.flatten
+    @ocns = @ht_items.first.ocns
 
-  def initialize(*htitems)
-    @htitems = htitems.flatten
-    @ocns = @htitems.first.ocns
-
-    if @htitems.find {|h| h.ocns != @ocns }
+    if @ht_items.find {|c| c.ocns != @ocns }
```

Comment:
For consistency this should probably call `Holding#batch_with?`. While something with the ocns `[2,1]` could be batched with ocns `[1,2]`, it isn't really a problem that it isn't, since the batching is just an optimization to being with.